mappings/net/minecraft/client/gui/hud/debug/DebugHud.mapping

```diff
@@ -1,49 +1,85 @@
-CLASS net/minecraft/unmapped/C_epxplegp net/minecraft/client/gui/hud/DebugHud
+CLASS net/minecraft/unmapped/C_epxplegp net/minecraft/client/gui/hud/debug/DebugHud
+	FIELD f_ccxylgcn frameRateSampleLog Lnet/minecraft/unmapped/C_fnzofwlw;
+	FIELD f_cdiumrmi pingSampleLog Lnet/minecraft/unmapped/C_fnzofwlw;
 	FIELD f_demcwxlr client Lnet/minecraft/unmapped/C_ayfeobid;
 	FIELD f_fjhfdzhz HEIGHT_MAP_TYPES Ljava/util/Map;
 	FIELD f_hdjainem fluidHit Lnet/minecraft/unmapped/C_eetoegzn;
+	FIELD f_jwptaljh showingCharts Z
```

Comment:
```suggestion
	FIELD f_jwptaljh showCharts Z
```

Reply:
This one is actually why I named them all `showing`.  It doesn't control whether they're shown, rather it's updated to reflect whether charts are to be shown.  But maybe that difference isn't very important... or maybe it means the others should be `show` with that being `showing`, or event `chartsShown` or something to make the difference more obvious.

Reply:
Is `showingFpsTpsCharts` a mistake then? It's checked in `render()`.
Or am I missunderstanding the distinction?

Reply:
I've changed, for now, everything but `showingCharts` to use `show` instead btw.  I think that's probably clearest... I don't think the distinction would be immediately obvious, but it would prompt someone to think "why are they named differently?" and, upon checking their usages, seeing that `showingCharts` is used slightly differently.  At least, I'd probably do that :p